website/snippets/_install-dbt-extension.md

```diff
@@ -122,37 +128,37 @@ steps to finish setting up your account (_Note: This is not required to use the
 Now that you have activated your dbt developer account, you can access features of the <Constant name="dbt_platform" />. You can also re-download your registration key using the steps outlined in [Register with an existing dbt account](#register-with-an-existing-dbt-account) if you need to set up the dbt extension on a new machine.
 
 ### Register with an existing dbt account 
-
 <!-- This anchor is linked from the VS Code registration page. Please do not change it -->
 
-If you already have a dbt account, you do not need to re-register to use the dbt extension. The dbt extension can authenticate with the dbt platform using a `dbt_cloud.yml` file. If this file is present in your `~/.dbt/` folder, then the registration flow will automatically attempt to use this
-file during registration. If you do not have a `~/.dbt/dbt_cloud.yml` file downloaded, refer to the following instructions:
+During the VS Code extension registration workflow it can detect if you already have a <Constant name="dbt_platform" /> account or previously created one in the past (no need to create a new one!). 
+- If you already have a <Constant name="dbt_platform" /> account, you will need to use the same account to connect the extension. 
+- If you've never created a <Constant name="dbt_platform"/> account, skip this section and follow the earlier [New user registration](#new-user-registration) steps.
+- The extension will use the `dbt_cloud.yml` file from your <Constant name="dbt_platform"/> account, which securely stores your <Constant name="dbt_platform"/> account credentials for authentication. You can download this file from your <Constant name="dbt_platform"/>'s **Account settings** page.
 
-<Expandable alt_header="For dbt accounts with Fusion enabled">
```

Comment:
@mirnawong1 in the OP, you say:
> changes from expandables to tabs
Did you change your mind on this? I like the section headers better than tabs so please keep as is!